.circleci/config.yml

```diff
@@ -34,17 +34,18 @@ jobs:
           paths:
               - "venv"
               - "node_modules"
+
+      - run:
+          name: npm run lint
+          command: npm run lint
+
```

Comment:
and `prettier --list-different` would be another `run` step here?

Reply:
We could do that. That will mean that, if you haven't formatted properly, the build will fail, right? In that case, we're still running Prettier here and enforcing formatting - wouldn't it be better/nicer/friendlier to, instead of having the build fail (and forcing users to run formatting themselves), auto-format it in this step?

Reply:
> That will mean that, if you haven't formatted properly, the build will fail, 

Correct, similar to the `lint` step.

> auto-format it in this step?

How would it get added to the code though? Would CircleCI commit it back to this branch? I'd rather that CI didn't make any changes to the code, that the user is forced to make the changes.

Reply:
I thought that code pushed to Github would pass through CircleCI first, so whatever changes happen in CircleCI would be reflected on Github. But now that I think about it, I guess that's not how it works. Ok! I'll add the `--list-different` command.

Reply:
Yeah, CircleCI just runs a copy of the code in the repo at each commit, CircleCI doesn't write back to GitHub at all (unless you program to do that explicitly)

Reply:
Yes and that would be too magical. I would expect the code I write to be the code that ends up in Github :)